apps/language_server/lib/language_server/experimental/code_mod/format.ex

```diff
@@ -44,15 +44,21 @@ defmodule ElixirLS.LanguageServer.Experimental.CodeMod.Format do
     |> formatter.()
   end
 
-  @spec formatter_for(String.t()) ::
+  @spec formatter_for(String.t(), String.t() | nil) ::
           {:ok, formatter_function, keyword()} | {:error, :no_formatter_available}
-  defp formatter_for(uri_or_path) do
+  defp formatter_for(uri_or_path, project_dir) do
     path = Conversions.ensure_path(uri_or_path)
 
     try do
       true = Code.ensure_loaded?(Mix.Tasks.Format)
 
-      if function_exported?(Mix.Tasks.Format, :formatter_for_file, 1) do
+      if project_dir && Version.match?(System.version(), ">= 1.15.0") do
```

Comment:
we can use `-dev` or `-rc.0` for now to make testing easier

Reply:
done